hexrd/imageseries/load/framecache.py

```diff
@@ -26,13 +32,25 @@ def __init__(self, fname, style='npz', **kwargs):
         self._framelist = []
         self._framelist_was_loaded = False
         self._load_framelist_lock = Lock()
+        # TODO extract style from filename ?
+        self._style = style.lower()
+
+        ncpus = multiprocessing.cpu_count()
+        self._max_workers = kwargs.get('max_workers', ncpus)
 
-        if style.lower() in ('yml', 'yaml', 'test'):
+        if self._style in ('yml', 'yaml', 'test'):
             self._from_yml = True
             self._load_yml()
-        else:
+        elif self._style == "npz":
             self._from_yml = False
             self._load_cache()
+        elif self._style == "fch5":
+            self._from_yml = False
+            self._load_cache()
+        else:
+            raise TypeError("Unknown style format for loading data: {style}."
```

Comment:
```suggestion
            raise TypeError(f"Unknown style format for loading data: {style}."
```

Because of {style}, it looks like this is supposed to be an f-string, but it's not. Might need to add f before the start of the string.